src/widgets/AppWindow.js

```diff
@@ -1212,10 +1234,10 @@ export default class AppWindow extends createjs.Container {
     });
     this._contentArea.accessible.addChild(complementary);
 
-    const sumaryText = 'SUMMARY: \n' +
-                        'Ishmael explains that, whenever he feels depressed and suicidal, he always goes to sea.\n' +
-                        'Ishmael claims that most people and most cultures around the world have a special attraction \n' +
-                        'to water in general and the sea in particular.';
+    const sumaryText = 'SUMMARY: \n'
+                        + 'Ishmael explains that, whenever he feels depressed and suicidal, he always goes to sea.\n'
+                        + 'Ishmael claims that most people and most cultures around the world have a special attraction \n'
+                        + 'to water in general and the sea in particular.';
```

Comment:
I'm surprised this indent level passes linting.  I would have thought that it could only increase 1 level for the additional lines instead of multiple.